docs/_events/header_menu_bar.md

```diff
@@ -0,0 +1,59 @@
+---
+name: Header menu bar
```

Comment:
To date I've used all lowercase for the names of events and attributes, partly because I was aiming for consistency and partly because I've built search interfaces based on the names of things and run afoul of capitalisation. Happy to discuss the merits of either approach, but I think we should be consistent.

Reply:
We could go all lowercase and use CSS to capitalise for readability.

Reply:
@andysellick Thanks - I've just realised that your `super_navigation_header.md` file is already documenting `header_menu_bar.md`. Which one should we keep? 😅 Does this mean that the file names should be their publishing component names instead of their `type` values?

Reply:
Argh! Hadn't noticed that. Good question 🤔 

I think for now let's merge your additions into the existing file, but let's have a chat soon about how we do this. To some extent the names of the files don't really matter, but maybe we want to give things more than one name so we can display them differently depending on filtering options? I've already given trackers a readable name and a technical name, for example.